src/ProbeMeasurements.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef WATER_PROBE_ACQUANATIVA_AP3_PROBE_MEASUREMENTS_HPP
+#define WATER_PROBE_ACQUANATIVA_AP3_PROBE_MEASUREMENTS_HPP
+
+#include <base/Float.hpp>
+#include <base/Temperature.hpp>
+#include <base/Time.hpp>
+
+namespace water_probe_acquanativa_ap3 {
+    struct ProbeMeasurements {
+        base::Time time;
+        
+        /** Water dissolved oxygen concentration in Kg/L */
+        float concentration = base::unknown<float>();
+
+        /** Water dissolved oxygen % saturation */
+        float saturation = base::unknown<float>();
```

Comment:
I liked the old names with `oxygen` in it, they were more descriptive
```suggestion
        /** Water dissolved oxygen concentration in Kg/L */
        float oxygen_concentration = base::unknown<float>();

        /** Water dissolved oxygen % saturation */
        float oxygen_saturation = base::unknown<float>();
```

Reply:
me too but Sylvain told me to change them to be more clean

Reply:
Did I ? I don't remember that. I actually agree with the suggestion ...

Reply:
yes, you did on monday.. I agree they're more descriptive.